ot/__init__.py

```diff
@@ -35,37 +35,41 @@
 from . import factored
 from . import solvers
 from . import gaussian
+from . import mapping
 
 # OT functions
-from .lp import (emd, emd2, emd_1d, emd2_1d, wasserstein_1d, 
-                binary_search_circle, wasserstein_circle, 
-                semidiscrete_wasserstein2_unif_circle)
+from .lp import (emd, emd2, emd_1d, emd2_1d, wasserstein_1d,
+                 binary_search_circle, wasserstein_circle,
+                 semidiscrete_wasserstein2_unif_circle)
 from .bregman import sinkhorn, sinkhorn2, barycenter
 from .unbalanced import (sinkhorn_unbalanced, barycenter_unbalanced,
                          sinkhorn_unbalanced2)
 from .da import sinkhorn_lpl1_mm
-from .sliced import (sliced_wasserstein_distance, max_sliced_wasserstein_distance, 
+from .sliced import (sliced_wasserstein_distance, max_sliced_wasserstein_distance,
                      sliced_wasserstein_sphere, sliced_wasserstein_sphere_unif)
 from .gromov import (gromov_wasserstein, gromov_wasserstein2,
-                        gromov_barycenters, fused_gromov_wasserstein, fused_gromov_wasserstein2)
+                     gromov_barycenters, fused_gromov_wasserstein, fused_gromov_wasserstein2)
 from .weak import weak_optimal_transport
 from .factored import factored_optimal_transport
 from .solvers import solve
+from .mapping import (nearest_brenier_potential_fit, nearest_brenier_potential_predict_bounds, joint_OT_mapping_kernel,
```

Comment:
Thanks for the review @rflamary!, The [last commit](https://github.com/PythonOT/POT/pull/526/commits/55a9b2836ff465c9490a028bc3b7b6d254b76ad9) addresses your comment, and fixes the example icon. I also made the function `ot.mapping._ssnb_qcqp_constants` private (added the `_`), since a user would have no use for this helper function on its own.